docker/Dockerfile

```diff
@@ -62,26 +63,29 @@ RUN export USE_CUDA=1
 
 ARG CUDA_VERSION=""
 
-RUN TORCH_VER=$(curl --silent --location https://pypi.org/pypi/torch/json | python -c "import sys, json, pkg_resources; releases = json.load(sys.stdin)['releases']; print(sorted(releases, key=pkg_resources.parse_version)[-1])") && \
-    TORCH_VISION_VER=$(curl --silent --location https://pypi.org/pypi/torchvision/json | python -c "import sys, json, pkg_resources; releases = json.load(sys.stdin)['releases']; print(sorted(releases, key=pkg_resources.parse_version)[-1])") && \
+RUN git clone https://github.com/pytorch/serve.git
+
+WORKDIR "serve"
+
+RUN \
     if echo "$BASE_IMAGE" | grep -q "cuda:"; then \
         # Install CUDA version specific binary when CUDA version is specified as a build arg
         if [ "$CUDA_VERSION" ]; then \
-            python -m pip install --no-cache-dir torch==$TORCH_VER+$CUDA_VERSION torchvision==$TORCH_VISION_VER+$CUDA_VERSION -f https://download.pytorch.org/whl/torch_stable.html; \
-        # Install the binary with the latest CUDA version support
+            python ./ts_scripts/install_dependencies.py --cuda $CUDA_VERSION; \
```

Comment:
To solve common_gpu.txt installation, you only need ref
https://github.com/pytorch/serve/blob/master/docker/Dockerfile#L75

Reply:
@lxning  There is a problem with the CPU image too. It doesn't have pynvml , torchaudio, torchtext